kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/storage/cloud/KieServerStateCloudRepository.java

```diff
@@ -104,8 +98,10 @@ public void store(@NotNull String serverId, @NotNull KieServerState kieServerSta
             ann.put(STATE_CHANGE_TIMESTAMP,
                     ZonedDateTime.now().format(DateTimeFormatter.ISO_INSTANT));
             
-            if (stateUpdated) {
+            if (this.isRolloutRequired) {
```

Comment:
Why the rollout is required just in some cases when Kie server state is stored?
I think that rollout is needed every time when a state is changed (and Kie server is completely initialized).

Reply:
See comment above pertaining KieServerStateRepository.store().